SPECS/kernel-hci/0022-update-kernel-version-5.15.48.100.patch

```diff
@@ -0,0 +1,26 @@
+From 0c191a4653665df412c733fc7cad90d89959778e Mon Sep 17 00:00:00 2001
```

Comment:
you don't need this patch. I have it only because I build it from Sources not with RPM patches